PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cognitiveservices/src/main/java/com/microsoft/azure/toolkit/intellij/cognitiveservices/components/CognitiveSubscriptionComboBox.java

```diff
@@ -10,15 +10,46 @@
 import com.microsoft.azure.toolkit.lib.auth.AzureAccount;
 import com.microsoft.azure.toolkit.lib.cognitiveservices.AzureCognitiveServices;
 import com.microsoft.azure.toolkit.lib.common.model.Subscription;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang.BooleanUtils;
+
+import javax.annotation.Nonnull;
+import java.util.*;
+import java.util.stream.Collectors;
 
 public class CognitiveSubscriptionComboBox extends SubscriptionComboBox {
+    private boolean listUnselectedSubscriptions = false;
+    private Map<Subscription, Boolean> status = new HashMap<>();
+
+    public CognitiveSubscriptionComboBox(boolean listUnselectedSubscriptions) {
```

Comment:
I tend to list all subscriptions if there are no selected, and add a label indicating whether OpenAI is enabled for it.